### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by inspecta**
In my code-review branch, I have added the HTML and CSS linter, README.md, styles.css, index.html and .gitignore files.
-The linter will format my HTML and CSS code.
-The index.html page contains the only header
-The styles.css contains the header styles 
-The .gitignore file contains the test.md file that we aren't tracking by git.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (koachcode) [inline .gitignore:1]: - Good work on the project so far, but you need to implement one necessary change. PLease add the node module into the `.gitignore` as it is a requirement in the project.

**Review by koachcode — CHANGES_REQUESTED** [review]
Hi @inspecta , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
you-are-near

## Highlights
- Good linter setup :+1: 
- Good readme :+1: 
- Good commits messages :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (inspecta) [reply .gitignore:1]: Thanks @Ginohmk. Do I need to create the node module directory or I can just node module into the .gitignore?

Comment (koachcode) [reply .gitignore:1]: Hi @inspecta Please just add the node module, to the `.gitignore`  so it becomes a second nature to add it to a ignore file, Also your project should have included a node module, never the less, I would suggest you just add it to git ignore and request for a review.

**Review by muneebulrehman — APPROVED** [review]
Hi @inspecta , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- Good follow-up with the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**